Firmware/mmu2.cpp

```diff
@@ -860,7 +885,7 @@ StepStatus MMU2::LogicStep(bool reportErrors) {
     case Finished:
         // At this point it is safe to trigger a runout and not interrupt the MMU protocol
         CheckFINDARunout();
-        break;
+        [[fallthrough]]; // let Finished be reported the same way like Processing
```

Comment:
Please verify this change on MK3S. It may break current implementation on this platform. MK4 may have a different workflow.

Reply:
@gudnimg Did you test this?

Reply:
No I don’t think I tested it specifically. We can drop this change if you think its risky.